### PR TITLE
Hotfix module loader

### PR DESCRIPTION
**Pull request by doktordirk**
this isn't helping with my problem, but shouldn't harm either (no new build since build is unchanged)

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -4,7 +4,6 @@ System.config({
   babelOptions: {
     "optional": [
       "runtime",
-      "optimisation.modules.system",
```
Comment (RWOverdijk) [inline config.js:7]: Why did you remove this?

File: config.js [hunk 2]
```diff
@@ -67,10 +65,16 @@ System.config({
       "aurelia-pal": "npm:aurelia-pal@1.0.0-beta.1.0.2",
       "core-js": "npm:core-js@1.2.6"
     },
+    "npm:aurelia-dependency-injection@1.0.0-beta.1.0.1": {
```
Comment (RWOverdijk) [inline config.js:61]: Fork. @1.0.0-beta.1 is still in the file, too.

**Comment by RWOverdijk** [comment]
Looks good, but the updates caused some forks :/

File: config.js (hunk 1 above)
Comment (doktordirk) [inline config.js:7]: leftover from previous try. didn't help. but i had a problem with that line somewhere else, 
...
had a quick look now, and i clearly should have left it there

File: config.js (hunk 2 above)
Comment (doktordirk) [inline config.js:61]: can't be helped (easiliy). aurelia-templating uses 1.0.1 and aurelia-framework 1.0.0.

Comment (RWOverdijk) [inline config.js:61]: Well... Hm. I don't want to introduce forks. Perhaps consult with the guys over at Gitter?

Comment (doktordirk) [inline config.js:61]: i looked an an old map. aurelia-api is the culprit now

Comment (RWOverdijk) [inline config.js:61]: Noted, and merged. :)
